clang-tidy-checks/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
instead of using `&&` everywhere, could you `set -e` here?
```suggestion
#!/usr/bin/env bash
set -e
```